app/models/external_work.rb

```diff
@@ -123,4 +123,8 @@ def creators
   def revised_at
     created_at
   end
+
+  def unrevealed?
+    false
+  end
```

Comment:
This is mostly a style thing, but to match the existing code for dealing with unrevealed bookmarkables it would be better to not add this method here. Instead, use `bookmarkable.try(:unrevealed?)` in the get_count_for_bookmark_blurb method to safely check the result of the method even if the object doesn't have it ([docs](https://api.rubyonrails.org/classes/Object.html#method-i-try))